spec/models/api/v1/student_registration_spec.rb

```diff
@@ -29,4 +29,33 @@
     its(:student) { should be_valid }
   end
 
-end      
\ No newline at end of file
+  describe "#save" do
+    context "given duplicate user names" do
+      it "creates unique logins" do
+
+        @count = User.all.size
```

Comment:
It won't mater too much in this case, but in Rails 3 it is more efficient to use `User.count`. That will do an SQL query to just get the number of users.  In Rails 3 `User.all` immediately executes a select query to get all of the records.  In Rails 4 this was changed to be lazy (it returns a Relation object instead of an array).